Source/WebKit/Shared/RemoteLayerTree/RemoteScrollingCoordinatorTransaction.cpp

```diff
@@ -219,6 +221,15 @@ static void dump(TextStream& ts, const ScrollingStateStickyNode& node, bool chan
         ts << node.viewportConstraints();
 }
 
+static void dump(TextStream& ts, const ScrollingStatePluginHostingNode& node, bool changedPropertiesOnly)
+{
```

Comment:
Did you mean to write?

```cpp
    dump(ts, static_cast<const ScrollingStateNode&>(node), changedPropertiesOnly);
```

Reply:
Hmm, I wonder what @smfr was on about with the `ScrollingStateFrameHostingNode` version (which this is a copy of). Will see if there's anything interesting in the output!

Reply:
I think there's just nothing to dump.